### PR TITLE
add border size as optional property for the radio button  

### PR DESCRIPTION
**Pull request by StefanLuecke**

### CODE REVIEW

**Comment by adhamhassan99** [comment]
This needs to be accepted it is much needed

File: lib/RadioButton.tsx [hunk 1]
```diff
@@ -17,8 +17,10 @@ export default function RadioButton({
   onPress,
   selected = false,
   size = 24,
+  borderSize
 }: RadioButtonProps) {
-  const borderWidth = PixelRatio.roundToNearestPixel(size * 0.1);
+
+  const borderWidth = borderSize ?? PixelRatio.roundToNearestPixel(size * 0.1);
```
Comment (ThakurBallary) [inline lib/RadioButton.tsx:23]:
```suggestion
  const borderWidth = PixelRatio.roundToNearestPixel(borderSize);
```

File: lib/RadioButton.tsx [hunk 2]
```diff
@@ -17,8 +17,10 @@ export default function RadioButton({
   onPress,
   selected = false,
   size = 24,
+  borderSize
```
Comment (ThakurBallary) [inline lib/RadioButton.tsx:20]:
```suggestion
  borderSize = 2,
```

File: README.md [hunk 3]
```diff
@@ -115,6 +115,7 @@ onPress | function | no |  | any function
 selected | boolean | no | false | true / false
 size | number | no | 24 | positive numbers
 value | string | no |  | any string
+borderSize | number | no | | positive numbers
```
Comment (ThakurBallary) [inline README.md:118]:
```suggestion
borderSize | number | 2 | | positive numbers
```

**Review by ThakurBallary — COMMENTED** [review]
add border size

**Comment by ThakurBallary** [comment]
@StefanLuecke and @adhamhassan99 thanks for writing. This is released in v2.3.0

**Pull request merged**